src/Webhook.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Pusher;
+
+class Webhook
+{
+    public $time_ms;
+    public $events = array();
+}
```

Comment:
I think it would be safer to make these private (with getters) and have them set through the constructor. At the moment it's possible to forget to set one, or both of these; having them as constructor params makes it more explicit that they are required.

Reply:
Yep, I should know better!